news.html

```diff
@@ -1,3 +1,12 @@
+<span>
```

Comment:
I'm confused why this diff is showing up, looking into it. The master already has the Dockstore blog in its branch. Looking into it.